### ISSUE TITLE
Blood sugars don’t show up

### ISSUE THREAD

**Issue by many7695** [nielsmaerten/glucose-ticker#47]
I stop using glucose ticker when heroku add a price, multiple months later I’m now using nightscoutpro 
![IMG_0660](https://user-images.githubusercontent.com/3122959/236647068-692e1420-e6bb-4f31-8531-ba2249c8b13c.jpeg)

instead of getting my sugars the -1, could it be because the default role is denied or is there something else going on

**Comment by nielsmaerten** [comment]
Yes, you're correct: If your default role does not have 'readable' access, you must first create a new token with at least 'readable' access.  
Then, when you enter the URL of your site, include the token like so:
![image](https://github.com/nielsmaerten/glucose-ticker/assets/4604406/da88927b-7afe-4e63-965f-e6e0606aae69)

**Comment by many7695** [comment]
Thanks for that it worked, when or if  u update glucose ticker can or is it possible to enable copy and paste on my windows 10 pc it wouldn’t let me to it

**Comment by many7695** [comment]
Pasting the url into glucose ticker

**Comment by nielsmaerten** [comment]
I'll look into it. For now, best to use CTRL+V to paste in the url, that should work fine.

**Comment by Jacobby147** [comment]
Hi 

This method with nightscout pro version does not work. Can you please show the exact URL with the token layout that will work for the widget? What else besides creating a role with a token and pasting it to the URL code after the database address should be done? 

![image](https://github.com/nielsmaerten/glucose-ticker/assets/111612645/c5ba02b8-ae84-4bb1-8692-fcb7c6bd4ccf)

**Comment by nielsmaerten** [comment]
Hey @Jacobby147 
As far as I can see, you've entered the URL and the token correctly.  

The exact URL should look like this:
`https://my.nightscout.site/?token=testrole-e981ffd2b898d82a`

I copied this directly from my Nightscout site by right-clicking the token, and then "Copy link" or "Copy link address" like so:
![image](https://github.com/nielsmaerten/glucose-ticker/assets/4604406/102e6dfb-1ecb-4f69-8c8f-348daac6d535)

But yours seems correct, so I'm not sure what's wrong. If restarting the app does not fix it, could you try getting the logs? You can do this by starting `C:\Users\yourname\AppData\Local\Programs\Glucose Ticker\glucose-ticker-base.exe` from a command terminal

![image](https://github.com/nielsmaerten/glucose-ticker/assets/4604406/f9aa6178-fa10-4d76-9921-3b14837e0f5e)
